### ISSUE TITLE
Feature: persist window docking state 

### ISSUE THREAD

**Issue by AlonAm** [AnyStatus/Support#73]
As a user, I would like to configure whether AnyStatus Desktop should start in "Docked" or "Undocked" mode.

**Comment by bobted** [comment]
In addition (or as a separate feature), when undocked, retain the position and size of the window. When the app is closed, then reopened, the main window should (optionally) show in the same location and size when it was last closed.

**Comment by AlonAm** [comment]
This feature is on the roadmap and will be implemented further down the road.
Thank you for your feedback!

**Comment by AlonAm** [comment]
There's also a bug which causes the window to hide when docked on certain screens. Will fix that too.

**Comment by AlonAm** [comment]
Released in AnyStatus Desktop 2.1.22

**Comment by bobted** [comment]
Just downloaded and installed. 

At first the window opened in the upper-left corner (of monitor 1) instead of the center. I moved the window to the lower-right corner of monitor 1 and made the window a bit wider and taller.

I have 12 repos, all use the Jenkins View configuration and all having at least 6 branches. I wanted to test the new feature so I tried to close AnyStatus by using the tray icon's context menu. It seems that the app doesn't close while in the middle of a refresh of the branches; at least during the initial scan of the branches after the app starts. Once the app closed, I reopened it, but it opened in the upper-left corner of monitor 1, again.

I then moved the moved back to the lower-right corner and resized it. This time, I docked it to the right side of monitor 1. I waited a few minutes to make sure the initial scan of the repos was finished before trying to close the app; this time it closed right away. When I opened the app again, it opened as being docked to the right side of monitor 1 as expected; when undocked, it went back to the lower-right corner of monitor 1 with the size that I had made it before docking.

So it seems the feature is working, but there might be a small timing issue with the initial repo scan and persisting the window size/state.

Thanks for implementing this.
